web/ui/module/codemirror-promql/src/complete/hybrid.ts

```diff
@@ -54,6 +52,8 @@ import {
   UnquotedLabelMatcher,
   QuotedLabelMatcher,
   QuotedLabelName,
+  AnotherKindOfDuration,
+  NumberDurationLiteral,
```

Comment:
I think you will have to adjust something about the highlighting as well? Because neither type of literal now gets syntax-highlighted correctly anymore (it just shows up in black instead of green-ish).

Reply:
indeed I missed this part. It is now fixed :)